spec/lib/open_food_network/feature_toggle_spec.rb

```diff
@@ -40,11 +40,19 @@ def stub_foo(value)
         end
       end
 
+      context 'and the feature is enabled for "all"' do
+        before { FeatureToggle.enable(:foo, ["all"]) }
+
+        it 'returns true' do
+          expect(FeatureToggle.enabled?(:foo, user)).to eq(true)
+        end
+      end
+
       context 'and the feature is disabled for them' do
         before { FeatureToggle.enable(:foo, []) }
 
         it 'returns false' do
-          expect(FeatureToggle.enabled?(:foo, user)).to eq(false)
+          expect(FeatureToggle.enabled?(:foo, user)).to eq(true)
```

Comment:
⚠️

Reply:
Is this WIP?

Reply:
it wasn't until I realized my brain wasn't at 100% :sweat: